### PR TITLE
APEXMALHAR-2460 Redshift output module unable to emit tuples.

### PR DESCRIPTION
**Pull request by deepak-narkhede**
Tested with newly created application kinesis-to-redshift app-template (under separate PR).

### CODE REVIEW

**Comment by deepak-narkhede** [comment]
@chaithu14 . Please review.

File: library/src/main/java/org/apache/apex/malhar/lib/fs/s3/S3Reconciler.java [hunk 1]
```diff
@@ -172,11 +172,11 @@ public void endWindow()
     while (doneTuples.peek() != null) {
       FSRecordCompactionOperator.OutputMetaData metaData = doneTuples.poll();
       removeIntermediateFiles(metaData);
-      /*if (outputPort.isConnected()) {
+      if (outputPort.isConnected()) {
```
Comment (chaithu14) [inline library/src/main/java/org/apache/apex/malhar/lib/fs/s3/S3Reconciler.java:177]: Please add the unit test.

Comment (deepak-narkhede) [reply library/src/main/java/org/apache/apex/malhar/lib/fs/s3/S3Reconciler.java:177]: @chaithu14 . Added the test.

**Comment by chaithu14** [comment]
Looks good to me.

**Review by chaithu14 — APPROVED** [review]

**Pull request merged**